terraform/lib/dependabot/terraform/registry_client.rb

```diff
@@ -80,7 +80,7 @@ def all_module_versions(identifier:)
         response = http_get!(URI.join(base_url, "#{identifier}/versions"))
 
         JSON.parse(response.body).
-          fetch("modules").first.fetch("versions").
+          fetch("modules").first.fetch("versions", []).
```

Comment:
Depending on what you're after, you could return the object you would expect to reference 👇 
```suggestion
          fetch("modules", [{}]).first.fetch("versions", []).
```

If this is something we shouldn't encounter/handle, you could raise an error 👇  

```ruby
  fetch("modules") { raise NoModulesError }
  .first
  .fetch("versions")

Reply:
Actually that doesn't work here because the `modules` property does exist, it's just an empty list. So I used `tap` to check for `nil`.